src/main/java/racingcar/model/RaceGame.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.dto.FinalResultDto;
+import racingcar.model.dto.RoundIndividualResultDto;
+import racingcar.model.dto.RoundResultDto;
+
+public class RaceGame {
+    private final RaceCars raceCars;
+    private final Movement movement;
+
+    public RaceGame(RaceCars raceCars, Movement movement) {
+        this.raceCars = raceCars;
+        this.movement = movement;
+    }
+
+    public void moveAllCars() {
+        raceCars.moveAllCars(movement);
+    }
+
+    public RoundResultDto getRoundResult() {
+        List<Car> cars = raceCars.getCars();
+        return new RoundResultDto(
+                cars.stream().map(car ->
+                                new RoundIndividualResultDto(car.getName().getValue(), car.getPosition().getValue())
+                        )
+                        .collect(Collectors.toList())
+        );
+    }
```

Comment:
stream으로 생성된 배열이 변경될 이유가 없다면 Collectors.toList() 대신 .toList()를 사용해봐도 좋을 것 같아요!

```suggestion
    public RoundResultDto getRoundResult() {
        List<Car> cars = raceCars.getCars();
        return new RoundResultDto(
                cars.stream().map(car ->
                                new RoundIndividualResultDto(car.getName().getValue(), car.getPosition().getValue())
                        )
                        .toList()
        );
    }
```

이렇게 수정하면 코드가 더 간결해지면서도 불변성을 자연스럽게 확보할 수 있을 것 같아요!

Reply:
반환 리스트가 변경의 여지가 없을 때는 toList()! -> 알고는 있었지만 습관적으로 코딩을 하다 보니 적용을 못했던 것 같습니다. 더 생각하면서 코드를 작성해야할 것 같네요 😅